app/code/core/Mage/Customer/Model/Convert/Parser/Customer.php

```diff
@@ -84,6 +84,7 @@ public function getCustomerModel()
      * Retrieve customer address model cache
      *
      * @return Mage_Customer_Model_Address|object
+     * @deprecated
```

Comment:
I've deprecated this method cause it's used only in this class (well, not anymore after this PR)